tests/integration/kubernetes/k8s-guest-pull-image.bats

```diff
@@ -13,6 +13,7 @@ setup() {
     setup_common 
     unencrypted_image_1="quay.io/sjenning/nginx:1.15-alpine"
     unencrypted_image_2="quay.io/prometheus/busybox:latest"
+    large_image="quay.io/confidential-containers/test-images:largeimage"
```

Comment:
We might want to have a discussion of whether to have this image hosted/build in kata-containers, but I think in the short-term it's fine.